relayer/__init__.py

```diff
@@ -7,17 +7,45 @@
 
 
 class Relayer(object):
+    """
+    Arguments:
+        logging_topic: desired kafka topic to send logs, this will get modified if a prefix o suffix is set.
+        context_handler_class: class that receives an emitter and expose at least a method emit and log.
+    Keyword Arguments:
+        kafka_hosts: 'host[:port]' string (or list of 'host[:port]'
+            strings) that the producer should contact to bootstrap initial
+            cluster metadata. This does not have to be the full node list.
+            It just needs to have at least one broker that will respond to a
+            Metadata API Request.
+        topic_prefix (str): value to prefix all topics handled by relayer.
+            Defaults to empty string.
+        topic_suffix (str): value to suffix all topics handled by relayer.
+            Defaults to empty string.
+        source: If defined it must be a json serializable value.
```

Comment:
However, I do agree that lacking defaults that hint the type or proper PEP 484 type hints, some effort to declare the type must be done, Which is already present by specifying the default type as in: `Defaults to empty string.`